windows/source/app.manifest

```diff
@@ -0,0 +1,76 @@
+﻿<?xml version="1.0" encoding="utf-8"?>

```

Comment:
@rmrlangford Don't we need to delete this file?

Reply:
@lauravignoli , nope, keeping it :)